### PR TITLE
MAGECLOUD-4925:  Ability to support custom database parameters in docker image

### PR DESCRIPTION
**Pull request by arhiopterecs**
## Purpose of this pull request

This pull request (PR) adds a description about customizing MariaDB settings

## Affected DevDocs pages

- https://devdocs.magento.com/cloud/docker/docker-containers-service.html#database-container

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/cloud/docker/docker-containers-service.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ When a database container initializes, it creates a new database with the specif
 
 To prevent accidental data loss, the database is stored in a persistent **`magento-db`** volume after you stop and remove the Docker configuration. The next time you use the `docker-compose up` command, the Docker environment restores your database from the persistent volume. You must manually destroy the database volume using the `docker volume rm <volume_name>` command.
 
-You can inject a MySQL configuration into the database container at creation by adding the configuration to the `docker-compose-override.yml` file. Add the custom values using an included `my.cnf` file, or add the correct variables directly to the override file as shown in the following examples.
+You can inject a MySQL configuration into the database container at creation by adding the configuration to the `docker-compose-override.yml` file. Add the custom values using an included `my.cnf` file like new mount or copy the custom configuration file to the directory `.docker/mysql/mariadb.conf.d`. Also, you can add the correct variables directly to the override file as shown in the following examples.
```
Comment (meker12) [inline src/cloud/docker/docker-containers-service.md]:
```suggestion
You can inject a MySQL configuration into the database container at creation by adding the configuration to the `docker-compose-override.yml` file. Add the custom values by including a `my.cnf` file using a mount, or by copying the custom configuration file to the directory `.docker/mysql/mariadb.conf.d`. Also, you can add the correct variables directly to the override file as shown in the following examples.
```

Comment (arhiopterecs) [reply src/cloud/docker/docker-containers-service.md]: Done. Thank You!

**Comment by meker12** [comment]
running tests

**Pull request merged**